openmetadata-spec/src/main/resources/json/schema/configuration/sslConfig.json

```
@@ -0,0 +1,15 @@
+{
+  "$id": "https://open-metadata.org/schema/entity/configuration/sslConfig.json",
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "title": "SSLConfig",
+  "description": "This schema defines the SSL Config.",
+  "type": "object",
+  "javaType": "org.openmetadata.api.configuration.airflow.SSLConfig",
+  "properties": {
+    "validate": {
+      "description": "Validate SSL",
+      "$ref": "../security/ssl/validateSSLClientConfig.json"
+    }
```

Do we need the "validate" field? Can not we do:
```suggestion
    "certificatePath": {
      "description": "CA certificate path. E.g., /path/to/public.cert",
      "type": "string"
    }
```

---

tried to be a bit generic in case there were other options we might want to handle in the future. E.g., this is now expecting a local cert path, maybe we want to be able to pick up certificates from S3/GCS at some point. Then we can add a new ref to `validateS3SSLClientConfig` etc

---

Fair enough 👍